deps/src/main/java/com/microsoft/azure/sdk/iot/deps/serializer/MethodParser.java

```diff
@@ -188,26 +188,26 @@ else if (jsonElement instanceof JsonObject)
                 {
                     if (statusTagNode == null)
                     {
-                        /**
-                         * Codes_SRS_METHODPARSER_21_010: [If the json contains any payload without `methodName` or `status` identification, the fromJson shall parse only the payload, and set the operation as `payload`]
-                         *  Ex:
-                         *  {
-                         *      "input1": "someInput",
-                         *      "input2": "anotherInput"
-                         *  }
+                        /*
+                          Codes_SRS_METHODPARSER_21_010: [If the json contains any payload without `methodName` or `status` identification, the fromJson shall parse only the payload, and set the operation as `payload`]
```

Comment:
You can remove this from what I know. @timtay-microsoft ?

Reply:
I'm wondering if we just need to remove the part that prefixes a code.

Reply:
I'd keep the sample json since it may be a useful reference, but yeah you can delete the  // Codes_SRS_ stuff